python/cudf/cudf/tests/test_udf_masked_ops.py

```diff
@@ -33,18 +38,32 @@ def test_arith_masked_vs_masked(op):
     # This test should test all the typing
     # and lowering for arithmetic ops between
     # two columns
-    def func_pdf(row):
+    def func(row):
         x = row["a"]
         y = row["b"]
         return op(x, y)
 
-    def func_gdf(row):
+    gdf = cudf.DataFrame({"a": [1, None, 3, None], "b": [4, 5, None, None]})
+    run_masked_udf_test(func, func, gdf, check_dtype=False)
```

Comment:
I think we could do it at this point yes. There were only two functions originally because one was a `@nulludf` decorated function so it couldn't be accepted by pandas. Now we can genuinely use the same unmodified function object.

Reply:
I think we could do it at this point yes. There were only two functions originally because one was a `@nulludf` decorated function so it couldn't be accepted by pandas. Now we can genuinely use the same unmodified function object.

Reply:
Was able to remove argument except `test_masked_udf_nested_function_support` because nested functions still takes a different func object - wrote tests inline instead.